internal/oci/container.go

```diff
@@ -269,9 +278,15 @@ func (c *Container) Metadata() *pb.ContainerMetadata {
 
 // State returns the state of the running container
 func (c *Container) State() *ContainerState {
```

Comment:
@tedyu can you break this change to a separate PR? I want us to land the changes in this PR in stages.

Reply:
I will keep Container#State() as is in this PR.

Reply:
@mrunalp 
Can you take another look ?

Reply:
@mrunalp
Do you have other comment ?